test/extension/handlers/lifecycle/handlers.go

```diff
@@ -62,14 +66,17 @@ func NewExtensionHandlers(client client.Client) *ExtensionHandlers {
 // NOTE: custom RuntimeExtension, must implement the body of this func according to the specific use case.
 func (m *ExtensionHandlers) DoBeforeClusterCreate(ctx context.Context, request *runtimehooksv1.BeforeClusterCreateRequest, response *runtimehooksv1.BeforeClusterCreateResponse) {
 	log := ctrl.LoggerFrom(ctx).WithValues("Cluster", klog.KObj(&request.Cluster))
+	ctrl.LoggerInto(ctx, log)
```

Comment:
```suggestion
	ctx = ctrl.LoggerInto(ctx, log)
```
Otherwise it doesn't do anything afaik

Reply:
Good catch 😂